### DISCUSSION TITLE
organised and optimised code for tailwind css

### DISCUSSION THREAD

**Discussion by 18121A05L2** [tailwindlabs/tailwindcss#9734] [Help]
Hello Guys , recently i am using tailwind and loved to use in future projects too ....

but i need to implement tailwind css class names to follow the coding standards for a better organised  code to maintain at later and also anyone who is reviewing the code could be able to understand it easily

i actually need few things 
1) i know one way which is writing class names in the tailwind config file by giving a single name for multiple class names , but if project increases the tailwind config file too will be big like thousands of lines

2) how to organise all the class names specific to the components in different files  or any better standard that many follows to maintain it later 

3) if u had any other things like how to organise tailwind css fully in the MERN Stack App , please help me with the tips

**Comment by wongjn** [comment]
1. The classes should be more "global" and not component-scoped – one of the main points of the framework is to promote the class reuse across your entire project so you should stick to one class name for per property-value-variant combination.
2. You should organize the components at the template level and not organize the CSS per component if your project stack allows. Again, as per the previous point, the Tailwind classes are more "global" and you then only need to maintain the markup as component templates, instead of markup **and** CSS. This also means there is a single source of truth – the component template.
3. No experience with MERN Stack.
